docker-compose.yml

```diff
@@ -8,13 +7,16 @@ services:
   database:
     image: postgres:15
     container_name: quackStack-db
+    env_file:
+      - path: ./.env
+        required: false
     environment:
-      - POSTGRES_USER=${POSTGRES_USER:-admin}
-      - POSTGRES_PASSWORD=${POSTGRES_PASSWORD:-admin}
-      - POSTGRES_DB=${POSTGRES_DATABASE:-admin}
-      - PORT=5432
+      POSTGRES_USER: ${POSTGRES_USER:-admin}
+      POSTGRES_PASSWORD: ${POSTGRES_PASSWD:-admin}
```

Comment:
"POSTGRES_PASSWD" doesn't match your template.
you should make your env's match whats in the compose to avoid confusion and having to cross reference files